docs(tutorials): esxi cluster dedibox (#114)
scaleway/docs-content: tutorials/create-esxi-cluster-dedibox/index.mdx

  <Lightbox src="scaleway-idrac_console.png" width="600" alt="" />

7. Click **Connect Virtual Media** in the menu on the top of the virtual console window. A pop-up displays.

8. Click **Choose file** (1) and select the downloaded VMware ESXi ISO file. Then click **Map device** (2) to connect the virtual CD-ROM drive to the server. Once done close the pop-up.

  <Lightbox src="scaleway-map_device.png" width="500" alt="" />

9. Reboot your machine from the Dedibox console.

10. Press **F11** on your keyboard, during the boot process, to enter the boot manager:

  <Lightbox src="scaleway-boot_manager.png" width="400" alt="" />

11. Click **One-shot BIOS Menu** in the boot manager:

  <Lightbox src="scaleway-one_shoot_boot.png" width="500" alt="" />

12. Select the **Virtual Optical Drive** to boot the machine from the connected ISO file:

  <Lightbox src="scaleway-virtual_drive.png" width="500" alt="" />

> 13. The server is booting from the ISO file. Depending on your Internet connection, this may take a while. Once ready, the ESXi installer displays. 

Reviewer: I think this shouldn't be a numbered step, since there is no action.

Author: ```suggestion
The server is booting from the ISO file. Depending on your Internet connection, this may take a while. Once ready, the ESXi installer displays. 
```

</Message>

<Message type="tip">

If you do not have a Windows Server 2016 ISO file, [download](https://www.microsoft.com/en-us/evalcenter/evaluate-windows-server-2016) it directly from Microsoft.

</Message>

### Creating a Virtual Machine

1. Connect to the ESXi web interface of one of the machines.

2. Click **Storage** > **Datastore browser** to browse the contents of your datastores.

3. Once you are at the desired location for storing your ISO files, click **Upload** and select the ISO file on your local computer. Confirm by clicking **OK**. The upload progress is visible in the top right corner of the datastore browser. Once the upload is completed, you can close the datastore browser.

  <Message type="note">

  Depending on the speed of your Internet connection, the upload may take a while.

  </Message>

  <Lightbox src="scaleway-datastore_browser.png" width="600" alt="" />

4. Create a new virtual machine by clicking on **Virtual Machines** > **Create / Register VM**

5. Select **Create a new virtual machine** and confirm by clicking **Next**.

> 6. Enter a name for the new virtual machine, select the compatability, guest OS family and guest OS version. Then tick the box to enable Windows Virtualization Based Security and click **Next** to proceed:

Reviewer 2: ```suggestion
6. Enter a name for the new virtual machine, select the compatibility, guest OS family and guest OS version. Then tick the box to enable Windows Virtualization Based Security and click **Next** to proceed:
```

17. Double click the downloaded ISO file to display it as a virtual CD-ROM drive in Windows. Double click the `autorun.exe` application to launch the installation wizard:

  <Lightbox src="scaleway-autorun.png" width="500" alt="" />

18. During the setup wizard, choose the embedded setup, which allows you to install all vCenter components on a single machine:

  <Lightbox src="scaleway-vcenter_embedded.png" width="500" alt="" />

19. Choose a system network name for your vCenter. This name must be a working FQDN:

  <Lightbox src="scaleway-vcenter_name.png" width="500" alt="" />

20. Configure the credentials for the single sign-on system of vCenter:

  <Lightbox src="scaleway-vcenter_signon.png" width="500" alt="" />

21. Set the vCenter service account. You can keep the default settings and continue by clicking **Next**:

  <Lightbox src="scaleway-vcenter_service_account.png" width="500" alt="" />

22. Configure the vCenter database. You can keep the default settings so it uses an internal Postgres database:

  <Lightbox src="scaleway-vcenter_database.png" width="500" alt="" />

> 23. The default ports used by the vCenter application are displayed. You can keep the default settings, and click **Next** to continue:

Reviewer: ```suggestion
23. View the default port information used by the vCenter application and click **Next** to continue.
```

  <Lightbox src="scaleway-vcenter_ports.png" width="500" alt="" />

24. Decide if you want to participate in the "Customer Experience Improvement Program" by sharing data with VMware. Untick the box if not, and click **Next** to continue

  <Lightbox src="scaleway-vcenter_cei.png" width="500" alt="" />

> 25. A summary of your configuration displays. Check again if all your settings are correct and click **Install** to launch the installation of VMware vCenter

Reviewer: Put " A summary of your configuration displays" as a seperate line (no number) after the image. Start this step with "Check again..."

  <Lightbox src="scaleway-vcenter_review.png" width="500" alt="" />

> 26. Once the installation is completed, you can log off from the Windows VM and type the address of your vCenter Server in your web browser to connect to it.

Reviewer: ```suggestion
26. Log out of the Windows VM once the installation completes, and type the address of your vCenter Server in your web browser to connect to it.
```
